### ISSUE TITLE
Unity

### ISSUE THREAD

**Issue by Scott12334** [felHR85/UsbSerial#248]
Hello,
Is there any way to make this work with unity?

**Comment by felHR85** [comment]
@Scott12334 
Sure there is a way but I don't really know

**Comment by Scott12334** [comment]
Ok thank you, I am doing it with Android Studio now and the installation does not seem to be working. Attached I have the gradle dependency and repository code. No errors when syncing with the rest of the project but UsbSerialDevice doesn't work. I am following this video: https://www.youtube.com/watch?v=QHa6HWTmQFs. Any ideas?
![2](https://user-images.githubusercontent.com/50891409/58328972-797fda80-7e01-11e9-977f-68c9045ff4c0.JPG)
![1](https://user-images.githubusercontent.com/50891409/58328973-797fda80-7e01-11e9-9966-8c8f74e6b831.JPG)

**Comment by kamran1993** [comment]
yes i use this library in unity3d first : you most write code in android studio and make a jar file 
but there is some tip on it , i commit my example project in github as soon as possible.

**Comment by felHR85** [comment]
@kamrankhorasani I would love to add a link to your example project.

**Comment by kamrankhorasani** [comment]
@felHR85 thank you Felipe , I use your library to a large project but I decide write a plug-in and example of your library that used in unity . So when I write this plug-in send message to you to add my link to your Page .

**Comment by Scott12334** [comment]
Could also send me that link?
Thanks
